### DISCUSSION TITLE
Timed out error on start code-server

### DISCUSSION THREAD

**Discussion by devarroyofernando** [coder/code-server#4119] [General]
I have a problem to run code-server in kubernets because timed out error. Run without problems in local, but I try in docker container on kubernets received this error.

I need increase the time to timed out, it's possible?

**Comment by jsjoeio** [comment]
Good question! I am not a Docker or Kubernetes expert but @bpmct or @code-asher might be able to help

**Reply by devarroyofernando** [reply]
Thks.

**Reply by code-asher** [reply]
No idea haha :sweat_smile: I have no experience with Kubernetes unfortunately.

**Comment by devarroyofernando** [comment]
Do you know if the only way to increase the 10 second timed out time is to redo the build?
I found a constant that determines this time in src/node/wrapper.js, but I am not able to build to test.

**Reply by jsjoeio** [reply]
@code-asher might know

**Reply by code-asher** [reply]
Ahh so the timeout is from code-server's wrapper.

The only way to change this is to rebuild.

You might have quite a bit of trouble down the line though if a process takes more than ten seconds to spawn.

**Reply by code-asher** [reply]
If you can edit the built code before starting code-server that should also work.

**Reply by devarroyofernando** [reply]
Thanks, I will try to change and rebuild.

**Reply by bpmct** [reply]
Let me know how this works! I can see if I can reproduce if not

**Comment by devarroyofernando** [comment]
I managed to rebuild but it is giving an error in GLIB 2.1.7 stating that version 2.2.5 is needed, in the version that was already built there was no error. I need to install vscode on centOS and I can't update GLIB. Would they be able to tell me which version is now required for version 2.2.5 or if there is something I can change to work on version 2.1.7?

**Reply by jsjoeio** [reply]
Does it mention argon or argon2 at all? That could be why it's asking for 2.2.5. 

> Would they be able to tell me which version is now required for version 2.2.5

Hmm...I can't recall exactly which version but 3.10 and above probably all require this new version. You could try 3.9.3

**Reply by code-asher** [reply]
The minimum version of glibc required will depend on the version you used to build it.

So the easiest way to handle this is to build on the same or similar machine you will deploy code-server so it uses the same libc (this is why we build code-server using centOS 7 as it has an old glibc).
